docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,23 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
+
+> In interfaces, functions in actual declarations can be non-abstract and have a
+> body. To indicate that in `expect` declarations, explicitly mark functions as `open`:
```

Comment:
"that" is a little bit unclear for me here: for example, I might assume that this is a way to give expect-functions bodies, while it is in fact only about changing their modality. I'd rephrase it in a more specific way, just as an example:

"In interfaces, functions in actual declarations can be non-abstract and have a body, which allows inheritors to not implement this function. `expect`-functions can not have bodies, so to let common-inheritors know that it's not necessary to implement particular function, one can mark it as `open`. Naturally, all `actual`-counterparts will be required to have bodies as well"

(still far from ideal IMO, I'm not that good at this stuff :) )